lambdas/issue-client-access-token/src/main/java/uk/gov/di/ipv/core/issueclientaccesstoken/domain/ConfigurationServicePublicKeySelector.java

```diff
@@ -49,32 +39,19 @@ public List<? extends PublicKey> selectPublicKeys(
             Context context)
             throws InvalidClientException {
 
-        JWSAlgorithm algorithm = jwsHeader.getAlgorithm();
-        String clientId = claimedClientID.getValue();
-        String publicKeyMaterial =
-                configService.getParameter(PUBLIC_KEY_MATERIAL_FOR_CORE_TO_VERIFY, clientId);
+        var clientId = claimedClientID.getValue();
+        var algorithm = jwsHeader.getAlgorithm();
+        if (algorithm != ES256) {
+            throw new InvalidClientException(
+                    String.format(
+                            "%s algorithm is not supported. Received from client ID '%s'",
+                            algorithm, clientId));
+        }
 
         try {
-            switch (algorithm.toString()) {
```

Comment:
Yeah, I think back in the day, before we were live, we were using RS256 from our stub. But then the real auth/orch (before the split) were going to use ES256 so we added support from it. Now we just use ES256.
I think we were using certs because that's what we were used to handling from Verify days. But try getting a signed X509 cert using KMS...